content/cloud/changelog/spin-cloud-v0-5-0.md

```diff
@@ -0,0 +1,25 @@
+title = "Spin Cloud v0.5 Updates"
+template = "changelog_item"
+date = "2023-11-02T12:00:00Z"
+enable_shortcodes = true
+tags = ["spin_cloud_plugin"]
+[extra]
+type= "changelog_post"
+twitter_card_type = "summary_large_image" 
+image = "/static/image/changelog/spin-cloud-0.5.0.jpg"
+
+---
+
+## Spin Cloud v0.5.0 Changelog
+
+We’re excited to announce the v0.5.0 release of the `spin cloud` plugin. Some notable improvements over the last minor update include: 
+
+- Support for component model on Fermyon Cloud! Simply use [Spin 2.0’s application manifest](https://developer.fermyon.com/spin/v2/manifest-reference) and run `spin cloud deploy` to deploy your component model enlightened Spin application to Fermyon Cloud. You can learn more about the many benefits of the component model (polyglot, performance, & portability, to name a few) with [our latest blog post](https://www.fermyon.com/blog/index).
```

Comment:
Is the plan to update this to the specific blog post when ready?

Reply:
Yes, just letting it settle on the top of the blog index, where the post will be sitting waiting. We can update here once another blog pushes it down.